Source/FullScreenPokemon.js

```diff
@@ -2584,6 +2593,19 @@ var FullScreenPokemon;
                 }
             }
         };
+        /**
+         * Activates a Detector to force the Player onto his/her bike and optionally to keep moving.
+         *
+         * @param player   The Player.
+         * @param thing   A Detector triggered by player.
+         */
+        FullScreenPokemon.prototype.activateCyclingRoad = function (player, thing) {
```

Comment:
How do you plan to allow the player to get back to a non-forced state? (leaving the cycling area?)